### ISSUE TITLE
Uncaught FileNotFoundError: calendar is not exists.


### ISSUE THREAD

**Issue by UrbanPlan-LDB** [MIERUNE/GTFS-GO#68]
Steps to reproduce
 1. First 
 2. Second 
 3. Third 

```
Traceback (most recent call last):
  File "C:\Users/Lucio De Benedictis/AppData/Roaming/QGIS/QGIS3\profiles\Lucio_02_23/python/plugins\GTFS-GO-master\gtfs_go_dialog.py", line 237, in execution
    gtfs_parser = GTFSParser(extracted_dir)
  File "C:\Users/Lucio De Benedictis/AppData/Roaming/QGIS/QGIS3\profiles\Lucio_02_23/python/plugins\GTFS-GO-master\gtfs_parser\__main__.py", line 35, in __init__
    self.dataframes = self.__load_tables(txts)
  File "C:\Users/Lucio De Benedictis/AppData/Roaming/QGIS/QGIS3\profiles\Lucio_02_23/python/plugins\GTFS-GO-master\gtfs_parser\__main__.py", line 64, in __load_tables
    raise FileNotFoundError(f"{datatype} is not exists.")
FileNotFoundError: calendar is not exists.

```
GTFS-GO-master 3.1.1, QGIS 3.28.4-Firenze on Windows 10

**Comment by kenseii** [comment]
Hey @UrbanPlan-LDB My guess would be that the issue is happening on a custom dataset because the calendar.txt file is missing, do you mind sharing that custom GTFS dataset for test?

**Comment by Kanahiro** [comment]
@UrbanPlan-LDB 
Hi, I apologize for late response.
The issue error occurs when calendar.txt is missing would be solved on `v3.2.3`, will be released in few days.
If not to be so, please add new issue with your dataset, thanks.
